UnrealPlugin/Source/PlayfabGSDK/PlayfabGSDK.Build.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright Epic Games, Inc. All Rights Reserved.
```

Comment:
Can we fix the copyright notice in some of the files? Thanks!

Reply:
fixed the license

Reply:
as mentioned, can we make it Apache license as the rest of the repo?